lib/mongoose.js

```diff
@@ -35,7 +35,7 @@ module.exports = function FilterablePlugin(Filterable, schema, options) {
             return [
                 field,
                 {
-                    type: _.contains(VALID_TYPES, schemaField.type)? schemaField.type : null,
+                    type: _.contains(VALID_TYPES, schemaField.instance)? schemaField.type : null,
```

Comment:
Understood. It wasn't clear from your README/Docs that you currently only support mongoose < `4.x`.

However, this is a mute issue now since I've withdrawn my PR.